MANIFEST.in

```diff
@@ -13,3 +13,5 @@ include CHANGES.rst
 include LICENSE
 include versioneer.py
 include niworkflows/_version.py
+
+exclude codecov.yml
```

Comment:
Excludes are to pare down over-broad includes. `codecov.yml` will not be included by default, so does not need to be excluded.

```
$ python setup.py sdist | grep codecov
warning: no files found matching 'niworkflows/data/tests/work/*'
warning: no previously-included files found matching 'codecov.yml'
```

Reply:
Okay, i've removed the exclude.